src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,103 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { findMovie } from '../../api/movies';
```

Comment:
looks like there is no such function `findMovie` and file in your PR. From where do you import it? 🤔

Reply:
Probably, you forgot to push all files?